device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O28/port_config.ini

```diff
@@ -0,0 +1,49 @@
+##
+## Copyright (c) 2020-2023 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
this is a new file thus license should be 2023 only.

Reply:
Fixed in https://github.com/sonic-net/sonic-buildimage/pull/16784/commits/64761e9ad3f008c9ed3fd1e657b6121622fcb174